src/pages/settings/Profile/CustomStatus/StatusPage.js

```diff
@@ -1,122 +1,195 @@
+                navigateBackToPreviousScreen();
+            });
+        },
+        [currentUserClearAfter, draftClearAfter, isValidClearAfterDate, navigateBackToPreviousScreen],
+    );
 
     const clearStatus = () => {
         User.clearCustomStatus();
-        User.clearDraftCustomStatus();
+        User.updateDraftCustomStatus({
+            text: '',
+            emojiCode: '',
+            clearAfter: DateUtils.getEndOfToday(),
+        });
+        formRef.current.resetForm({[INPUT_IDS.EMOJI_CODE]: initialEmoji});
     };
 
-    const navigateBackToSettingsPage = useCallback(() => {
-        const topMostReportID = Navigation.getTopmostReportId();
-        if (topMostReportID) {
-            Navigation.goBack(ROUTES.REPORT_WITH_ID.getRoute(topMostReportID));
+    useEffect(() => setBrickRoadIndicator(isValidClearAfterDate() ? null : CONST.BRICK_ROAD_INDICATOR_STATUS.ERROR), [isValidClearAfterDate]);
+
+    useEffect(() => {
+        if (!currentUserEmojiCode && !currentUserClearAfter && !draftClearAfter) {
+            User.updateDraftCustomStatus({clearAfter: DateUtils.getEndOfToday()});
         } else {
-            Navigation.goBack(ROUTES.SETTINGS_PROFILE, false, true);
+            User.updateDraftCustomStatus({clearAfter: currentUserClearAfter});
         }
-    }, []);
 
-    const updateStatus = useCallback(() => {
-        User.updateCustomStatus({text: defaultText, emojiCode: defaultEmoji});
-
-        User.clearDraftCustomStatus();
-        Navigation.goBack(ROUTES.SETTINGS_PROFILE);
-    }, [defaultText, defaultEmoji]);
-    const footerComponent = useMemo(
-        () =>
-            hasDraftStatus ? (
-                <Button
-                    success
-                    text={translate('statusPage.save')}
-                    onPress={updateStatus}
-                />
-            ) : null,
-        [hasDraftStatus, translate, updateStatus],
-    );
+        return () => User.clearDraftCustomStatus();
+        // eslint-disable-next-line react-hooks/exhaustive-deps
+    }, []);
 
-    useEffect(() => () => User.clearDraftCustomStatus(), []);
+    const validateForm = useCallback(() => {
+        if (brickRoadIndicator) {
+            return {clearAfter: ''};
+        }
+        return {};
+    }, [brickRoadIndicator]);
 
     return (
-        <HeaderPageLayout
-            title={translate('statusPage.status')}
-            onBackButtonPress={navigateBackToSettingsPage}
-            headerContent={
-                <MobileBackgroundImage
-                    pointerEvents="none"
-                    style={styles.staticHeaderImage}
-                />
-            }
-            headerContainerStyles={[styles.staticHeaderImage]}
-            backgroundColor={theme.PAGE_THEMES[SCREENS.SETTINGS.PROFILE.STATUS].backgroundColor}
-            footer={footerComponent}
+        <ScreenWrapper
+            style={[StyleUtils.getBackgroundColorStyle(theme.PAGE_THEMES[SCREENS.SETTINGS.PROFILE.STATUS].backgroundColor)]}
```

Comment:
Any reason for adding this style?

This caused style regression on light mode:
<img width="374" alt="Screenshot 2023-12-14" src="https://github.com/Expensify/App/assets/108292595/fce08bd4-e698-4ed8-ab41-2c4087c4943f">

Reply:
This part was added before theme switching. So needs to be updated now